pkg/adapter/gcppubsub/adapter.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	eventType = "google.pubsub.topic.publish"
+	eventType = "dev.knative.event.pubsub.topic"
```

Comment:
I actually think we should leave it as google.pubsub.topic.publish since it's provided by google. You could see additional ones being created for other cloud providers and so forth?